src/index.js

```diff
@@ -16,7 +16,7 @@ export syntax interface = ctx => {
   let items = matchInterfaceItems(inner);
   let fields = items.reduce((acc, item) => {
     if (item.type === 'field' || item.type === 'static') {
-      return acc.concat(#`${item.name}: { value: Symbol(),
+      return acc.concat(#`${item.name}: { value: Symbol(${JSON.stringify(`${name}.${item.name}`)}),
```

Comment:
Good call but Sweet's API is a bit low level right now and won't auto-cast a string value to a string literal token. You need to do:

```js
Symbol(${fromStringLiteral(item.name, unwrap(name).value + '.' + unwrap(item.name).value)})
```

(I'd normally use a template literal there but there's a bug I just discovered sweet-js/sweet-core#710 with function calls in template literals that I'm very surprised we hadn't run into before)